README.rst

```diff
@@ -57,12 +57,12 @@ Compiling from source
 
 The Walrus has the following dependencies:
 
-* `Python <http://python.org/>`_ >= 3.7
-* `NumPy <http://numpy.org/>`_  >= 1.19.2
-* `Numba <https://numba.pydata.org/>`_ >= 0.49.1
-* `SciPy <https://scipy.org/>`_ >=1.2.1
-* `SymPy <https://www.sympy.org/>`_ >=1.5.1
-* `Dask[delayed] <https://docs.dask.org/>`_
+* `Python <http://python.org/>`_ >= 3.12
```

Comment:
does this mean `thewalrus` does not work with python 3.11 any more?

Reply:
I could expand the matrix to do more tests. I think we can be confident that it works with 3.12, but I do not believe we did any testing between 3.9 and 3.12.

Reply:
It works on `ubuntu-latest` with `3.10`, `3.11`, and `3.12`.

Reply:
Tests are also very close to passing on `windows-latest`, but there's a single failure regarding a `nan`.

Reply:
Got it!